downstream/titles/catalog/getting-started/master.adoc

```diff
@@ -158,11 +158,11 @@ The *Catalog Administrator* can add new users to existing groups. Once added, us
 You can add users to a group in the *Groups* view in *User Access*.
 
 . Navigate to *Settings* ➔ *User Access*.
-. Click *Groups*.
+. Click btn:[Groups].
 . Select a group from the list.
-. Click *Members* and then click *Add Member*.
+. Click btn:[Members] and then click btn:[Add Member].
```

Comment:
I'm not %100 certain whether these are buttons but would need to see the UI to be certain.

Reply:
Leaving this comment for now while we await review of the UI.